### DISCUSSION TITLE
Ability to combine PDFs

### DISCUSSION THREAD

**Discussion by SteveDevDev** [jonaswinkler/paperless-ng#969] [Feature Requests]
Sometimes I notice multiple PDFs that are really part of the same documentation.  Would be nice to be able to combine them into 1 document

**Comment by shamoon** [comment]
Hi @steveaggie , this is already in progress: https://github.com/jonaswinkler/paperless-ng/issues/335

**Reply by jonaswinkler** [reply]
And almost done.

* Need to test it more
* Error handling
* A couple issues regarding metadata handling

But overall, this is pretty much done.
